tests/pbft_manager_test.cpp

```diff
@@ -64,7 +64,7 @@ void check_2tPlus1_validVotingPlayers_activePlayers_threshold(size_t committee_s
     trxs_count++;
     EXPECT_HAPPENS({120s, 1s}, [&](auto &ctx) {
       for (auto &node : nodes) {
-        if (ctx.fail_if(!node->getFinalChain()->isKnownTransaction(trx.getHash()))) {
+        if (ctx.fail_if(!node->getFinalChain()->transaction_location(trx.getHash()))) {
```

Comment:
I believer we are using camel case for function names

Reply:
agree that the naming convention should be consistent. we have established naming for some types of members, but not for functions. I hope this is not critical to you right now, and anyway it should be fixed at once with a sweeping change.

Reply:
Really ? Because when I joined the project I was told that variables are underscore, class names and functions are camel-case

Reply:
Agreed offline that we don't aim at function naming consistency at this point, please resolve @JakubFornadel